### ISSUE TITLE
hippocampal microinfarct

### ISSUE THREAD

**Issue by sarahstofel** [mgijax/mammalian-phenotype-ontology#3421]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

hippocampal microfinfarct

>2. Synonyms (e.g., absent spleen)

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

an infarct located in the structure of the cerebrum involved with memory storage and spatial navigation that can only be observed microscopically 

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

https://jamanetwork.com/journals/jamaneurology/fullarticle/776636
DOI: 10.1001/archneur.57.6.869

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

cerebral microinfarct, abnormal hippocampus morphology 

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

https://orcid.org/0000-0003-2307-1226

**Comment by sbello** [comment]
As hippocampus is defined in the parent "abnormal hippocampus morphology" I just used hippocampus in the definition of this term rather than embedding the definition of hippocampus in the definition.
MP:0021006
